modules/rpc/handler_test.go

```diff
@@ -116,3 +116,32 @@ func (f fakeOnewayHandler) HandleOneway(ctx context.Context, p *transport.Reques
 	assert.NotNil(f.t, ctx)
 	return errors.New("oneway handle")
 }
+
+func TestInboundMiddleware_panic(t *testing.T) {
+	defer testPanicHandler(t)
```

Comment:
ya, then maybe we should apply parallel for all tests in this file. Let me pull another PR to address that. I do want to do some refactor as well.